pydocx/export/html.py

```diff
@@ -22,6 +22,30 @@
 
 class PyDocXHTMLExporter(PyDocXExporter):
 
+    def __init__(self, *args, **kwargs):
+        super(PyDocXHTMLExporter, self).__init__(*args, **kwargs)
+        self.heading_level_conversion_map = {
+            'heading 1': 'h1',
+            'heading 2': 'h2',
+            'heading 3': 'h3',
+            'heading 4': 'h4',
+            'heading 5': 'h5',
+            'heading 6': 'h6',
+        }
+        self.default_heading_level = 'h6'
+
+    def load_document(self):
+        document = super(PyDocXHTMLExporter, self).load_document()
```

Comment:
The base class assigns this to `self._document`. Does that matter? Looks like the `document` property might not cache the result, otherwise.